server/command.go

```diff
@@ -57,6 +57,17 @@ func (p *Plugin) hasSysadminRole(userID string) (bool, error) {
 	return true, nil
 }
 
+func (p *Plugin) hasChannelAdminRole(channelID string, userID string) (bool, error) {
+	channel, err := p.API.GetChannel(channelID)
+	if err != nil {
+		return false, err
+	}
+	if channel.CreatorId != userID {
+		return false, nil
+	}
```

Comment:
This check only allows the creator of a channel to run the clash command. The request in the ticket was to limit it to channel admins.

Reply:
We discussed this with @maisnamrajusingh in a meeting this morning. There is an API available for him to use https://developers.mattermost.com/extend/plugins/server/reference/#API.HasPermissionToChannel

Reply:
working on it guys, you can expect an update tomorrow

Reply:
This conversation is still unresolved.